todo_app/templates/index.html

```diff
@@ -18,16 +18,16 @@ <h2 class="text-center" style="margin: 40px 0 20px">Tasks</h2>
     <div class="col-lg">
       <h5 class="text-center">Open</h5>
       <ul class="list-group mb-4">
-      {% if not uncompleted_items %}
-      {% if completed_items %}
+      {% if not model.uncompleted_items %}
+      {% if model.completed_items %}
```

Comment:
It's a bit hard to follow if the blocks are not indented, but dw, you don't need to fix it